### DISCUSSION TITLE
Folders? Tags? Most Recent?

### DISCUSSION THREAD

**Discussion by khrome83** [pocketbase/pocketbase#3455] [Q&A]
Is there a way to have collections sorted and organized, besides the alphabetical collection name?
I have 20 collections already, and they fall into various themes. It would be great to have some organization.

Here are a few ideas - 

### Folders
One idea for folders would be to allow a name like `folder_name/collection_name` in the creation, which would display the UI item in a folder. Or it could be an additional field. 

Ideally these folders could be collapsed and expanded, and remember their state between sessions (local storage). 

The heading from settings (system, sync), would be the ideal visual with a icon to show collapsed or expanded.

### Tags
Another idea would be to add the concept of tags, where a collection can have 1 or more tags to document elements of it. This would need to be another field in the collection ui, and would be more expensive to build.

A filter can then be added next to search, that opens a flyout, allow for checking/unchecking tags, which then changes what is shown in the second column.

This is maybe more advanced, but not my favorite. Would require a lot more build effort.

### Most Recent
Perhaps the most simple solution. Record in local storage the number of times a collection had attention.
* Edited the Collection
* Viewed the Collection (clicked Secondary Menu element)
* Added Item to the Collection (via UI)
* Edited Item in the Collection (via UI)

Could be as simple as a hash map, { collection_key: count, other_collection_key: count}. 
This could get more advanced with the count being made of a running log that ages out (TTL), also stored in Local Storage. 

Then the top 3-4 of these could have a duplicate entry at the top of the second column (below search) under a header of "Most Used". Another header would be below that read "All Collections". 

The idea here is that it would surface the most frequent interacted with collections mitigating the need to search. The other benefit of this idea, is that it's only UI layer and requires no DB storage, table modification, or Form changes.

Thoughts? Any interest? Just some ideas to make it easier to manage the backend of applications as they grow.

**Comment by ganigeorgiev** [comment]
For now there are no plans for folder or tags as I think it will clutter the interface, but I've added in my todo to consider eventually adding something similar to your "Most recent" suggestion since it is a common request (_or maybe instead replace it with "pinned" collections as suggested in https://github.com/pocketbase/pocketbase/issues/3112; I'll have to experiment to see which option would make most sense_).

**Comment by fr3j** [comment]
Would still highly appreciate this feature in Pocketbase! loving it so far except for this, losing quite a bit of overview when having 14 collections with more to come. I think having a dropdown just like how system collections are filtered would be perfect and not cluttering at all
